src/core/reference/src/op/istft.cpp

```diff
@@ -0,0 +1,136 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/reference/istft.hpp"
+
+#include <algorithm>
+#include <functional>
+#include <vector>
+
+#include "openvino/core/shape.hpp"
+#include "openvino/reference/fft.hpp"
```

Comment:
Include for reference/fft.hpp may not be needed.

Reply:
Removed